packages/sync/src/modules/Full_Sync.php

```diff
@@ -700,4 +764,57 @@ private function read_option( $name, $default = null ) {
 
 		return $value;
 	}
+
+	/**
+	 * Prefix of the blog lock transient.
+	 *
+	 * @access public
+	 *
+	 * @var string
+	 */
+	const ENQUEUE_LOCK_TRANSIENT_PREFIX = 'jp_sync_enqueue_lock_';
+	/**
+	 * Lifetime of the blog lock transient.
+	 *
+	 * @access public
+	 *
+	 * @var int
+	 */
+	const ENQUEUE_LOCK_TRANSIENT_EXPIRY = 15; // Seconds.
```

Comment:
Should this be a setting?

Reply:
I like Settings, going to make it a Setting